easybuild/easyconfigs/g/goatools/goatools-1.1.6-foss-2020b.eb

```diff
@@ -0,0 +1,71 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'goatools'
+version = '1.1.6'
+
+homepage = 'https://github.com/tanghaibao/goatools'
+description = "A Python library for Gene Ontology analyses"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+# extra downloaded files are needed acc. to docs: https://github.com/tanghaibao/goatools#installation
+sources = [
+    {'source_urls': ['https://github.com/tanghaibao/goatools/archive/'],
+     'filename': 'v%(version)s.tar.gz'},
+    {'source_urls': ['http://geneontology.org/ontology/'],
+     'filename': 'go-basic.obo', 'extract_cmd': 'cp %s %(builddir)s'},
```

Comment:
The installation of this easyconfig is already failing because `go-basic.odo` has changed, and hence the checksum validation fails.

It's probably not a good idea to include these files in the installation itself, there's no way to ensure that this keeps working.

It's fine to let the end user download these files before using this tool, no?

Reply:
oh, didn't expect that to happen...you are right i guess, but we can't perform the complex sanity check in that case

Reply:
We can, we just have to include the downloading of the necessary files in the sanity check itself (which is better than not doing a sanity check at all imho)

I'm working on that, will open a PR once it's fully working...

Reply:
fixed in #13629

Reply:
thank you! ill cleanup&close then